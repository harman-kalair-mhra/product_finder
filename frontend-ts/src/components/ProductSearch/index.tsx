import React from "react";
import styled from "styled-components";

const StyledH1 = styled.h1`
  color: blue;
  font-size: 80px;
`;

export default function ProductSearch() {
  return (
    <form>
      {" "}
      <label>
        SEARCH:
        <input type="text" name="products" id="products" />
      </label>
      <input type="submit" value="SEARCH" />
    </form>
  );
}
