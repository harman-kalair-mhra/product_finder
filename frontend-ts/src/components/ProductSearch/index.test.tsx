import React from "react";
import { shallow, mount } from "enzyme";

import ProductSearch from "./index";

describe("Product Select", () => {
  it("renders shallow", () => {
    const component = shallow(<ProductSearch />);
    expect(component).toMatchSnapshot();
  }),
    it("renders mount", () => {
      const component = mount(<ProductSearch />);
      expect(component).toMatchSnapshot();
    });
});
