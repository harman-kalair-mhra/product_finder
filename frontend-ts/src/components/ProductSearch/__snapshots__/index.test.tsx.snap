// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Search renders mount 1`] = `
<ProductSearch>
  <div>
    <form
      onSubmit={[Function]}
    >
       
      <label>
        SEARCH:
        <input
          id="products"
          name="products"
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <input
        type="submit"
        value="SEARCH"
      />
    </form>
    <div>
      <h2>
        Searched Products
      </h2>
    </div>
  </div>
</ProductSearch>
`;

exports[`Product Search renders shallow 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
     
    <label>
      SEARCH:
      <input
        id="products"
        name="products"
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <input
      type="submit"
      value="SEARCH"
    />
  </form>
  <div>
    <h2>
      Searched Products
    </h2>
  </div>
</div>
`;

exports[`Product Select renders mount 1`] = `
<ProductSearch>
  <div>
    <form
      onSubmit={[Function]}
    >
       
      <label>
        SEARCH:
        <input
          id="products"
          name="products"
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <input
        type="submit"
        value="SEARCH"
      />
    </form>
    <div>
      <h2>
        Searched Products
      </h2>
    </div>
  </div>
</ProductSearch>
`;

exports[`Product Select renders shallow 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
     
    <label>
      SEARCH:
      <input
        id="products"
        name="products"
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <input
      type="submit"
      value="SEARCH"
    />
  </form>
  <div>
    <h2>
      Searched Products
    </h2>
  </div>
</div>
`;
